src/libraries/System.Diagnostics.EventLog/src/System.Diagnostics.EventLog.csproj

```diff
@@ -128,6 +128,9 @@
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Threading.AccessControl\src\System.Threading.AccessControl.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)-windows'">
```

Comment:
We usually don't use the `-os` portion for these conditions.

Reply:
Changed to !$(TargetFramework.StartsWith('$(NetCoreAppCurrent)'))